sql/2022/interop/layer_adoption.sql

```diff
@@ -0,0 +1,48 @@
+#standardSQL
```

Comment:
How come for the CSS chapter you used a RegEx (faster ?) and here you use `walkSelectors`?

Reply:
I mention in the comment for the CSS query that the parser doesn't extract `@layer` natively, so regex was more straightforward.

However this chapter has a few metrics looking at `@`-rules so I tried a new general purpose solution. `walkSelectors` is slightly better because it ensures we're testing the regex on only the selectors, and it includes inline style blocks.

No reason we shouldn't do this for CSS as well, besides being more work 😁